app/assets/stylesheets/media.scss

```diff
@@ -421,3 +421,31 @@
     }
   }
 }
+
+.video-js .vjs-text-track-settings {
+  background: white !important;
+  /* overflow: auto; */
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #000;
+  max-height: 375px;
+  max-width: 640px;
+  border-radius: var(--modal-border-radius);
+  box-shadow: 5px 10px 5px gray;
+  font-size: 1em;
+
+}
+
+.vjs-track-settings-controls button {
+  border: 1px solid black;
+  border-radius: var(--modal-border-radius);
+  box-shadow: 2px 3px 3px gray;
```

Comment:
I believe this is the only button in the site that has a shadow.  Do we want this to be so unique?

Reply:
I could totally remove that, I was just trying to make it a bit more obviously a button.